TUTORIAL.rst

```diff
@@ -0,0 +1,187 @@
+PDFmorph Tutorial
+#################
+
+Welcome! This will be a quick tutorial to accquaint users with PDFmorph
+and some of what it can do.
+
+As we described in the README and installation instructions, please make
+sure that you are familiar with working with your command line terminal
+before using this application.
+
+Before you've started this tutorial, please ensure that you've installed
+all necessary software and dependencies.
+
+Basic PDFmorph Workflow
+-----------------------
+
+	1. Open your Terminal or Command Prompt.
+
+	2. It it's not active already, activate your PDFmorph-equipped
+	   conda environment by typing in ::
+
+		source activate <pdfmorph_env>
+
+	   on Linux or ``activate <pdfmorph_env>`` on Windows.
+
+		* If you need to list your available conda environments,
+		  run the command ``conda info --envs`` or
+		  ``conda env list``
+
+        * Run the ``pdfmorph --help`` command and read over the
+          info on that page for a brief overview of some of what we will
+          explore in this tutorial.
+
+	3. Using the ``mkdir`` command, create a directory where you'll
+        store the tutorial PDF files and use the ``cd`` command to change
+        into that directory. You can download the tutorial files
+        :download:`here <tests/testdata/tutorialData.zip`. Then, ``cd``
+        into the ``tutorialData`` directory.
+
+        * The files in this dataset were collected by Soham Banerjee
+          at Brookhaven National Laboratory in Upton, New York.
+
+        * The files are PDF data collected on Iridium Telluride with
+          20% Rhodium Doping (IrRhTe2) with the first file (01) collected
+          at 10K and the last (44) at 300K. The samples increase in
+          temperature as their numbers increase. The "C" in their names
+          indicates that they have undergone cooling.
```

Comment:
This requires more explanation.  What does "undergone cooling" mean.

Has this been reviewed by @sbanerjee.  This should happen before me.....